build.rs

```diff
@@ -1,29 +1,31 @@
-use std::process::Command;
-use std::env;
+extern crate cc;
+extern crate glob;
 
 #[cfg(not(feature = "orangepi"))]
-const TARGET: &'static str = "wiringpi";
+const TARGET: &'static str = "wiringPi";
 #[cfg(feature = "orangepi")]
-const TARGET: &'static str = "wiringop";
+const TARGET: &'static str = "WiringOP";
 
 fn main() {
     if cfg!(feature = "development") {
         return;
     }
-    if let Ok(target) = env::var("TARGET") {
-        if !target.starts_with("arm-") {
-            return;
-        }
-    }
 
-    let out_dir = env::var("OUT_DIR").unwrap();
-    match Command::new("make").arg("-e").arg(TARGET).status() {
-        Ok(status) if !status.success() => panic!(
-            "failed to build wiringPi C library (exit code {:?})",
-            status.code()
-        ),
-        Err(e) => panic!("failed to build wiringPi C library: {}", e),
-        _ => {}
+    // only build wiringpi/wiringop for arm/armv7 platforms
+
+    let target = std::env::var("TARGET").unwrap();
+    if !(target.starts_with("arm-") || target.starts_with("armv7-")) {
```

Comment:
I wonder if it would be fine to just check for `arm` without the `-` or `v7-`... :thinking: